addons/repair/models/stock_picking.py

```diff
@@ -14,10 +14,13 @@ class PickingType(models.Model):
         ('repair_operation', 'Repair')
     ], ondelete={'repair_operation': 'cascade'})
 
+    count_repair_confirmed = fields.Integer(
+        string="Number of Repair Orders Confirmed", compute='_compute_count_repair')
+    count_repair_under_repair = fields.Integer(
+        string="Number of Repair Orders Under Repair", compute='_compute_count_repair')
+
     count_repair_ready = fields.Integer(
         string="Number of Repair Orders to Process", compute='_compute_count_repair')
-    count_repair_waiting = fields.Integer(
-        string="Number of Repair Orders Waiting", compute='_compute_count_repair')
     count_repair_late = fields.Integer(
         string="Number of Repair Orders Late", compute='_compute_count_repair')
```

Comment:
Not sure it's still required to keep the count since we remove it from the kanban view